Traceback (most recent call last):
  File "app.py", line 2, in <module>
    from wtforms import Form, TextAreaField, validators
ImportError: No module named wtforms
 * Running on http://0.0.0.0:5000/ (Press CTRL+C to quit)
 * Restarting with stat
 * Debugger is active!
 * Debugger pin code: 190-383-704
10.141.226.105 - - [04/Sep/2016 01:21:09] "GET /? HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 01:21:14] "POST /results HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/flask/app.py", line 2000, in __call__
    return self.wsgi_app(environ, start_response)
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/flask/app.py", line 1991, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/flask/app.py", line 1567, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/flask/app.py", line 1988, in wsgi_app
    response = self.full_dispatch_request()
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/flask/app.py", line 1641, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/flask/app.py", line 1544, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/flask/app.py", line 1639, in full_dispatch_request
    rv = self.dispatch_request()
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/flask/app.py", line 1625, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/scidb/flask/tf_app/app.py", line 64, in results
    y = classify(smi)
  File "/home/scidb/flask/tf_app/app.py", line 36, in classify
    d = Dataset.from_numpy(data_dir,fp,o,tasks=kinase_tasks)
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/deepchem/datasets/__init__.py", line 303, in from_numpy
    verbosity=verbosity)
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/deepchem/datasets/__init__.py", line 47, in __init__
    Dataset.write_data_to_disk(self.data_dir, "data", tasks, X, y, w, ids))
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/deepchem/datasets/__init__.py", line 126, in write_data_to_disk
    save_to_disk(X, os.path.join(data_dir, out_X))
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/deepchem/utils/save.py", line 27, in save_to_disk
    joblib.dump(dataset, filename, compress=compress)
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/sklearn/externals/joblib/numpy_pickle.py", line 404, in dump
    pickler.close()
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/sklearn/externals/joblib/numpy_pickle.py", line 284, in close
    with open(self._filename, 'wb') as zfile:
IOError: [Errno 13] Permission denied: u'data/data-X.joblib'
10.141.226.105 - - [04/Sep/2016 01:21:14] "GET /results?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 404 -
10.141.226.105 - - [04/Sep/2016 01:21:14] "GET /results?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 404 -
10.141.226.105 - - [04/Sep/2016 01:21:14] "GET /results?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 404 -
 * Running on http://0.0.0.0:5000/ (Press CTRL+C to quit)
 * Restarting with stat
 * Debugger is active!
 * Debugger pin code: 719-501-035
10.141.226.105 - - [04/Sep/2016 01:22:20] "POST /results HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 01:22:31] "GET /static/style.css HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 01:22:35] "GET /? HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 01:23:01] "POST /results HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 01:23:45] "GET /? HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 11:33:03] "GET /? HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/flask/app.py", line 2000, in __call__
    return self.wsgi_app(environ, start_response)
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/flask/app.py", line 1991, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/flask/app.py", line 1567, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/flask/app.py", line 1988, in wsgi_app
    response = self.full_dispatch_request()
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/flask/app.py", line 1641, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/flask/app.py", line 1544, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/flask/app.py", line 1639, in full_dispatch_request
    rv = self.dispatch_request()
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/flask/app.py", line 1625, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/scidb/flask/tf_app/app.py", line 57, in index
    return render_template('reviewform.html', form=form)
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/flask/templating.py", line 133, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/jinja2/environment.py", line 851, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/jinja2/environment.py", line 812, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/jinja2/environment.py", line 786, in _load_template
    template = self.loader.load(self, name, globals)
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/jinja2/loaders.py", line 125, in load
    code = environment.compile(source, name, filename)
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/jinja2/environment.py", line 565, in compile
    self.handle_exception(exc_info, source_hint=source_hint)
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/jinja2/environment.py", line 754, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/scidb/flask/tf_app/templates/reviewform.html", line 9, in template
    <link rel="shortcut icon" href="{{ url_for('static', filename='favicon.ico' }}">
TemplateSyntaxError: unexpected '}', expected ')'
10.141.226.105 - - [04/Sep/2016 11:33:04] "GET /?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 404 -
10.141.226.105 - - [04/Sep/2016 11:33:04] "GET /?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 404 -
10.141.226.105 - - [04/Sep/2016 11:33:04] "GET /?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 404 -
10.141.226.105 - - [04/Sep/2016 11:33:04] "GET /?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 404 -
10.141.226.105 - - [04/Sep/2016 11:33:46] "GET /? HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 11:33:47] "GET /static/css/bootstrap.min.css HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 11:33:47] "GET /static/favicon.ico HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 11:34:07] "POST /results HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 11:35:47] "GET /? HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 11:56:54] "GET /? HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 11:56:54] "GET /static/bitmap.png HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 11:56:54] "GET /static/css/bootstrap.min.css HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 11:56:54] "GET /static/fonts/glyphicons-halflings-regular.woff2 HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 11:56:54] "GET /static/favicon.ico HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 11:57:28] "GET /? HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 11:57:28] "GET /static/bitmap.png HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 11:57:28] "GET /static/css/bootstrap.min.css HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 11:57:28] "GET /static/fonts/glyphicons-halflings-regular.woff2 HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 11:57:28] "GET /static/favicon.ico HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 12:06:05] "GET /static/css/bootstrap.min.css.map HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 12:07:06] "GET /? HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 12:07:07] "GET /static/css/bootstrap.min.css HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:07:07] "GET /static/bitmap.png HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:07:07] "GET /static/fonts/glyphicons-halflings-regular.woff2 HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:12:12] "GET /? HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 12:12:13] "GET /static/css/bootstrap.min.css HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:12:13] "GET /static/bitmap.png HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:12:13] "GET /static/fonts/glyphicons-halflings-regular.woff2 HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:12:27] "GET / HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 12:17:15] "GET / HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 12:17:16] "GET /static/css/bootstrap.min.css HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:17:16] "GET /static/bitmap.png HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:17:16] "GET /static/fonts/glyphicons-halflings-regular.woff2 HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:21:41] "GET / HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 12:21:41] "GET /static/css/bootstrap.min.css HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:21:41] "GET /static/bitmap.png HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:21:41] "GET /static/fonts/glyphicons-halflings-regular.woff2 HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:24:06] "GET / HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 12:26:10] "GET / HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 12:26:10] "GET /static/css/bootstrap.min.css HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:26:10] "GET /static/bitmap.png HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:26:10] "GET /static/fonts/glyphicons-halflings-regular.woff2 HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:26:10] "GET /static/favicon.ico HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 12:32:12] "GET / HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 12:32:12] "GET /static/css/bootstrap.min.css HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:32:13] "GET /static/bitmap.png HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:32:13] "GET /static/fonts/glyphicons-halflings-regular.woff2 HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:33:42] "GET / HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 12:33:42] "GET /static/css/bootstrap.min.css HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:33:42] "GET /static/bitmap.png HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:33:42] "GET /static/fonts/glyphicons-halflings-regular.woff2 HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:36:34] "GET / HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 12:36:34] "GET /static/bitmap.png HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:36:34] "GET /static/css/bootstrap.min.css HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:36:34] "GET /static/fonts/glyphicons-halflings-regular.woff2 HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:39:36] "GET / HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 12:39:36] "GET /static/css/bootstrap.min.css HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:39:36] "GET /static/bitmap.png HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:39:36] "GET /static/fonts/glyphicons-halflings-regular.woff2 HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:42:55] "GET / HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 12:42:56] "GET /static/css/bootstrap.min.css HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:42:56] "GET /static/bitmap.png HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:42:56] "GET /static/fonts/glyphicons-halflings-regular.woff2 HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:44:11] "GET / HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 12:44:12] "GET /static/bitmap.png HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:44:12] "GET /static/css/bootstrap.min.css HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:44:12] "GET /static/fonts/glyphicons-halflings-regular.woff2 HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:45:07] "GET / HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 12:45:08] "GET /static/bitmap.png HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:45:08] "GET /static/css/bootstrap.min.css HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:45:08] "GET /static/fonts/glyphicons-halflings-regular.woff2 HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:46:07] "GET / HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 12:46:07] "GET /static/bitmap.png HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:46:07] "GET /static/css/bootstrap.min.css HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:46:07] "GET /static/fonts/glyphicons-halflings-regular.woff2 HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:47:25] "GET / HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 12:47:25] "GET /static/css/bootstrap.min.css HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:47:25] "GET /static/bitmap.png HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:47:25] "GET /static/fonts/glyphicons-halflings-regular.woff2 HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:49:28] "GET / HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 12:49:28] "GET /static/bitmap.png HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:49:28] "GET /static/css/bootstrap.min.css HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:49:28] "GET /static/fonts/glyphicons-halflings-regular.woff2 HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:49:40] "GET / HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 12:49:40] "GET /static/bitmap.png HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:49:40] "GET /static/css/bootstrap.min.css HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:49:40] "GET /static/fonts/glyphicons-halflings-regular.woff2 HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:49:49] "GET / HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 12:49:50] "GET /static/css/bootstrap.min.css HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:49:50] "GET /static/bitmap.png HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:49:50] "GET /static/fonts/glyphicons-halflings-regular.woff2 HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:50:23] "GET / HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 12:51:05] "GET / HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 12:51:05] "GET /static/css/bootstrap.min.css HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:51:06] "GET /static/bitmap.png HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:51:06] "GET /static/fonts/glyphicons-halflings-regular.woff2 HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:52:53] "GET / HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 12:52:54] "GET /static/bitmap.png HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:52:54] "GET /static/css/bootstrap.min.css HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:52:54] "GET /static/fonts/glyphicons-halflings-regular.woff2 HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:56:02] "GET / HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 12:56:02] "GET /static/css/bootstrap.min.css HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:56:02] "GET /static/bitmap.png HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:56:03] "GET /static/fonts/glyphicons-halflings-regular.woff2 HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:59:35] "GET / HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 12:59:36] "GET /static/bitmap.png HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:59:36] "GET /static/css/bootstrap.min.css HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 12:59:36] "GET /static/fonts/glyphicons-halflings-regular.woff2 HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 13:00:22] "GET / HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 13:00:23] "GET /static/css/bootstrap.min.css HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 13:00:23] "GET /static/bitmap.png HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 13:00:23] "GET /static/fonts/glyphicons-halflings-regular.woff2 HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 13:01:24] "GET / HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 13:01:24] "GET /static/css/bootstrap.min.css HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 13:01:24] "GET /static/bitmap.png HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 13:01:24] "GET /static/fonts/glyphicons-halflings-regular.woff2 HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 13:02:56] "GET / HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 13:02:56] "GET /static/bitmap.png HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 13:02:56] "GET /static/css/bootstrap.min.css HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 13:02:56] "GET /static/fonts/glyphicons-halflings-regular.woff2 HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 13:04:06] "GET / HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 13:04:06] "GET /static/bitmap.png HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 13:04:06] "GET /static/css/bootstrap.min.css HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 13:04:06] "GET /static/fonts/glyphicons-halflings-regular.woff2 HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 13:04:14] "GET / HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 13:04:14] "GET /static/css/bootstrap.min.css HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 13:04:14] "GET /static/bitmap.png HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 13:04:15] "GET /static/fonts/glyphicons-halflings-regular.woff2 HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 13:05:18] "GET / HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 13:05:19] "GET /static/css/bootstrap.min.css HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 13:05:19] "GET /static/bitmap.png HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 13:05:51] "POST /results HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 13:11:48] "POST /results HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 13:13:01] "POST /results HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 13:13:02] "GET /static/favicon.ico HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 13:15:09] "POST /results HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 13:15:52] "GET /static/favicon.ico HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 13:16:19] "POST /results HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 13:18:20] "POST /results HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 13:18:21] "GET /static/favicon.ico HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 13:18:31] "GET /? HTTP/1.1" 200 -
[13:18:50] SMILES Parse Error: extra open parentheses for input: 'CC(C(C)C(=O)N1CCCC1CNc2ccc3ncc(c4ccc(CN)cc4)n3n2'
10.141.226.105 - - [04/Sep/2016 13:18:50] "POST /results HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/flask/app.py", line 2000, in __call__
    return self.wsgi_app(environ, start_response)
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/flask/app.py", line 1991, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/flask/app.py", line 1567, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/flask/app.py", line 1988, in wsgi_app
    response = self.full_dispatch_request()
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/flask/app.py", line 1641, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/flask/app.py", line 1544, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/flask/app.py", line 1639, in full_dispatch_request
    rv = self.dispatch_request()
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/flask/app.py", line 1625, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/scidb/flask/tf_app/app.py", line 64, in results
    y = classify(smi)
  File "/home/scidb/flask/tf_app/app.py", line 33, in classify
    fp = AllChem.GetMorganFingerprintAsBitVect(mol,2,nBits=1024)
ArgumentError: Python argument types in
    rdkit.Chem.rdMolDescriptors.GetMorganFingerprintAsBitVect(NoneType, int)
did not match C++ signature:
    GetMorganFingerprintAsBitVect(RDKit::ROMol mol, int radius, unsigned int nBits=2048, boost::python::api::object invariants=[], boost::python::api::object fromAtoms=[], bool useChirality=False, bool useBondTypes=True, bool useFeatures=False, boost::python::api::object bitInfo=None)
10.141.226.105 - - [04/Sep/2016 13:18:51] "GET /results?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 404 -
10.141.226.105 - - [04/Sep/2016 13:18:51] "GET /results?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 404 -
10.141.226.105 - - [04/Sep/2016 13:18:51] "GET /results?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 404 -
10.141.226.105 - - [04/Sep/2016 13:18:51] "GET /results?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 404 -
10.141.226.105 - - [04/Sep/2016 13:19:05] "POST /results HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 13:23:00] "POST /results HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 13:23:00] "GET /? HTTP/1.1" 200 -
[13:23:38] SMILES Parse Error: extra open parentheses for input: 'CC(C(C)C(=O)N1CCN(CC1)c2ccc3ncc(c4ccccc4)n3n2'
10.141.226.105 - - [04/Sep/2016 13:23:38] "POST /results HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/flask/app.py", line 2000, in __call__
    return self.wsgi_app(environ, start_response)
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/flask/app.py", line 1991, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/flask/app.py", line 1567, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/flask/app.py", line 1988, in wsgi_app
    response = self.full_dispatch_request()
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/flask/app.py", line 1641, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/flask/app.py", line 1544, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/flask/app.py", line 1639, in full_dispatch_request
    rv = self.dispatch_request()
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/flask/app.py", line 1625, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/scidb/flask/tf_app/app.py", line 64, in results
    y = classify(smi)
  File "/home/scidb/flask/tf_app/app.py", line 33, in classify
    fp = AllChem.GetMorganFingerprintAsBitVect(mol,2,nBits=1024)
ArgumentError: Python argument types in
    rdkit.Chem.rdMolDescriptors.GetMorganFingerprintAsBitVect(NoneType, int)
did not match C++ signature:
    GetMorganFingerprintAsBitVect(RDKit::ROMol mol, int radius, unsigned int nBits=2048, boost::python::api::object invariants=[], boost::python::api::object fromAtoms=[], bool useChirality=False, bool useBondTypes=True, bool useFeatures=False, boost::python::api::object bitInfo=None)
10.141.226.105 - - [04/Sep/2016 13:23:39] "GET /results?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 404 -
10.141.226.105 - - [04/Sep/2016 13:23:39] "GET /results?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 404 -
10.141.226.105 - - [04/Sep/2016 13:23:39] "GET /results?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 404 -
10.141.226.105 - - [04/Sep/2016 13:24:03] "POST /results HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 13:27:40] "POST /results HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 13:29:44] "GET /? HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 13:29:50] "POST /results HTTP/1.1" 200 -
 * Detected change in '/home/scidb/flask/tf_app/app.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger pin code: 719-501-035
10.141.226.105 - - [04/Sep/2016 14:15:20] "GET /? HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 14:20:02] "GET /? HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 14:20:03] "GET /static/css/bootstrap.min.css HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 14:20:03] "GET /static/bitmap.png HTTP/1.1" 304 -
10.141.226.105 - - [04/Sep/2016 14:20:03] "GET /static/favicon.ico HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 14:20:20] "POST /results HTTP/1.1" 200 -
 * Detected change in '/home/scidb/flask/tf_app/app.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger pin code: 719-501-035
10.141.226.105 - - [04/Sep/2016 14:23:55] "GET /? HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 14:24:00] "POST /results HTTP/1.1" 200 -
 * Detected change in '/home/scidb/flask/tf_app/app.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger pin code: 719-501-035
10.141.226.105 - - [04/Sep/2016 14:24:31] "POST /results HTTP/1.1" 200 -
 * Detected change in '/home/scidb/flask/tf_app/app.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger pin code: 719-501-035
 * Detected change in '/home/scidb/flask/tf_app/app.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger pin code: 719-501-035
10.141.226.105 - - [04/Sep/2016 14:26:08] "POST /results HTTP/1.1" 200 -
 * Detected change in '/home/scidb/flask/tf_app/app.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger pin code: 719-501-035
10.141.226.105 - - [04/Sep/2016 14:27:47] "GET /? HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 14:28:01] "POST /results HTTP/1.1" 200 -
 * Detected change in '/home/scidb/flask/tf_app/app.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger pin code: 719-501-035
10.141.226.105 - - [04/Sep/2016 14:30:38] "GET /? HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 14:30:42] "POST /results HTTP/1.1" 200 -
 * Detected change in '/home/scidb/flask/tf_app/app.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger pin code: 719-501-035
10.141.226.105 - - [04/Sep/2016 14:31:27] "POST /results HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 14:31:28] "GET /static/favicon.ico HTTP/1.1" 200 -
 * Detected change in '/home/scidb/flask/tf_app/app.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger pin code: 719-501-035
10.141.226.105 - - [04/Sep/2016 14:37:36] "POST /results HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 14:37:37] "GET /static/favicon.ico HTTP/1.1" 200 -
 * Detected change in '/home/scidb/flask/tf_app/app.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger pin code: 719-501-035
10.141.226.105 - - [04/Sep/2016 14:45:36] "POST /results HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 14:45:36] "GET /static/favicon.ico HTTP/1.1" 200 -
 * Detected change in '/home/scidb/flask/tf_app/app.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger pin code: 719-501-035
10.141.226.105 - - [04/Sep/2016 14:47:51] "GET /? HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 14:47:55] "POST /results HTTP/1.1" 200 -
 * Detected change in '/home/scidb/flask/tf_app/app.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger pin code: 719-501-035
10.141.226.105 - - [04/Sep/2016 14:48:43] "POST /results HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 14:48:44] "GET /static/favicon.ico HTTP/1.1" 200 -
 * Detected change in '/home/scidb/flask/tf_app/app.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger pin code: 719-501-035
10.141.226.105 - - [04/Sep/2016 14:50:21] "POST /results HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 14:50:22] "GET /static/favicon.ico HTTP/1.1" 200 -
 * Detected change in '/home/scidb/flask/tf_app/app.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger pin code: 719-501-035
10.141.226.105 - - [04/Sep/2016 15:11:57] "POST /results HTTP/1.1" 200 -
10.141.226.105 - - [04/Sep/2016 15:12:06] "GET /download HTTP/1.1" 405 -
 * Detected change in '/home/scidb/flask/tf_app/app.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger pin code: 719-501-035
10.141.226.105 - - [04/Sep/2016 15:16:04] "GET /download HTTP/1.1" 400 -
 * Detected change in '/home/scidb/flask/tf_app/app.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger pin code: 719-501-035
10.141.226.105 - - [04/Sep/2016 15:16:29] "GET /download HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/flask/app.py", line 2000, in __call__
    return self.wsgi_app(environ, start_response)
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/flask/app.py", line 1991, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/flask/app.py", line 1567, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/flask/app.py", line 1988, in wsgi_app
    response = self.full_dispatch_request()
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/flask/app.py", line 1641, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/flask/app.py", line 1544, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/flask/app.py", line 1639, in full_dispatch_request
    rv = self.dispatch_request()
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/flask/app.py", line 1625, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/scidb/flask/tf_app/app.py", line 89, in download
    response = make_response(prediction)
NameError: global name 'make_response' is not defined
10.141.226.105 - - [04/Sep/2016 15:16:29] "GET /download?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 404 -
10.141.226.105 - - [04/Sep/2016 15:16:29] "GET /download?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 404 -
10.141.226.105 - - [04/Sep/2016 15:16:29] "GET /download?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 404 -
10.141.226.105 - - [04/Sep/2016 15:16:29] "GET /download?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 404 -
 * Detected change in '/home/scidb/flask/tf_app/app.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger pin code: 719-501-035
10.141.226.105 - - [04/Sep/2016 15:17:03] "GET /download HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/flask/app.py", line 2000, in __call__
    return self.wsgi_app(environ, start_response)
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/flask/app.py", line 1991, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/flask/app.py", line 1567, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/flask/app.py", line 1988, in wsgi_app
    response = self.full_dispatch_request()
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/flask/app.py", line 1641, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/flask/app.py", line 1544, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/flask/app.py", line 1639, in full_dispatch_request
    rv = self.dispatch_request()
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/flask/app.py", line 1625, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/scidb/flask/tf_app/app.py", line 89, in download
    response = make_response(prediction)
NameError: global name 'prediction' is not defined
10.141.226.105 - - [04/Sep/2016 15:17:04] "GET /download?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 404 -
10.141.226.105 - - [04/Sep/2016 15:17:04] "GET /download?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 404 -
10.141.226.105 - - [04/Sep/2016 15:17:04] "GET /download?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 404 -
10.141.226.105 - - [04/Sep/2016 15:17:04] "GET /download?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 404 -
10.141.226.105 - - [04/Sep/2016 15:35:34] "GET /download HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/flask/app.py", line 2000, in __call__
    return self.wsgi_app(environ, start_response)
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/flask/app.py", line 1991, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/flask/app.py", line 1567, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/flask/app.py", line 1988, in wsgi_app
    response = self.full_dispatch_request()
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/flask/app.py", line 1641, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/flask/app.py", line 1544, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/flask/app.py", line 1639, in full_dispatch_request
    rv = self.dispatch_request()
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/flask/app.py", line 1625, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/scidb/flask/tf_app/app.py", line 89, in download
    response = make_response(prediction)
NameError: global name 'prediction' is not defined
10.141.226.105 - - [04/Sep/2016 15:35:34] "GET /download?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 404 -
10.141.226.105 - - [04/Sep/2016 15:35:34] "GET /download?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 404 -
10.141.226.105 - - [04/Sep/2016 15:35:34] "GET /download?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 404 -
10.141.226.105 - - [04/Sep/2016 15:35:50] "POST /results HTTP/1.1" 200 -
Exception in thread Thread-1:
Traceback (most recent call last):
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/threading.py", line 801, in __bootstrap_inner
    self.run()
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/threading.py", line 754, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/werkzeug/serving.py", line 659, in inner
    srv.serve_forever()
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/werkzeug/serving.py", line 499, in serve_forever
    HTTPServer.serve_forever(self)
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/SocketServer.py", line 238, in serve_forever
    self._handle_request_noblock()
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/SocketServer.py", line 297, in _handle_request_noblock
    self.handle_error(request, client_address)
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/SocketServer.py", line 295, in _handle_request_noblock
    self.process_request(request, client_address)
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/SocketServer.py", line 321, in process_request
    self.finish_request(request, client_address)
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/SocketServer.py", line 334, in finish_request
    self.RequestHandlerClass(request, client_address, self)
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/SocketServer.py", line 655, in __init__
    self.handle()
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/werkzeug/serving.py", line 216, in handle
    rv = BaseHTTPRequestHandler.handle(self)
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/BaseHTTPServer.py", line 340, in handle
    self.handle_one_request()
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/werkzeug/serving.py", line 251, in handle_one_request
    return self.run_wsgi()
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/werkzeug/serving.py", line 193, in run_wsgi
    execute(self.server.app)
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/werkzeug/serving.py", line 184, in execute
    write(data)
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/werkzeug/serving.py", line 165, in write
    self.wfile.write(data)
IOError: [Errno 32] Broken pipe

 * Detected change in '/home/scidb/flask/tf_app/app.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger pin code: 719-501-035
10.141.226.105 - - [04/Sep/2016 16:07:07] "POST /results HTTP/1.1" 200 -
Exception in thread Thread-1:
Traceback (most recent call last):
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/threading.py", line 801, in __bootstrap_inner
    self.run()
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/threading.py", line 754, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/werkzeug/serving.py", line 659, in inner
    srv.serve_forever()
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/werkzeug/serving.py", line 499, in serve_forever
    HTTPServer.serve_forever(self)
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/SocketServer.py", line 238, in serve_forever
    self._handle_request_noblock()
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/SocketServer.py", line 297, in _handle_request_noblock
    self.handle_error(request, client_address)
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/SocketServer.py", line 295, in _handle_request_noblock
    self.process_request(request, client_address)
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/SocketServer.py", line 321, in process_request
    self.finish_request(request, client_address)
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/SocketServer.py", line 334, in finish_request
    self.RequestHandlerClass(request, client_address, self)
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/SocketServer.py", line 655, in __init__
    self.handle()
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/werkzeug/serving.py", line 216, in handle
    rv = BaseHTTPRequestHandler.handle(self)
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/BaseHTTPServer.py", line 340, in handle
    self.handle_one_request()
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/werkzeug/serving.py", line 251, in handle_one_request
    return self.run_wsgi()
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/werkzeug/serving.py", line 193, in run_wsgi
    execute(self.server.app)
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/werkzeug/serving.py", line 184, in execute
    write(data)
  File "/root/anaconda2/envs/my-rdkit-env/lib/python2.7/site-packages/werkzeug/serving.py", line 165, in write
    self.wfile.write(data)
IOError: [Errno 32] Broken pipe

